Apps/LabRecorder/CMakeLists.txt

```diff
@@ -58,23 +58,41 @@ add_executable(${target}
     targetver.h
 )
 
-IF(MSVC)
-    find_package(Boost REQUIRED COMPONENTS filesystem iostreams thread zlib)
-    set(zlib_libs Boost::zlib)
-ELSE()
-    find_package(Boost REQUIRED COMPONENTS filesystem iostreams thread)
-    find_package(ZLIB REQUIRED)
-    set(zlib_libs ${ZLIB_LIBRARIES})
-ENDIF()
+set(Boost_NO_SYSTEM_PATHS true)
+set (Boost_USE_STATIC_LIBS OFF CACHE BOOL "use static libraries from Boost")
+set (Boost_USE_MULTITHREADED ON)
+set(Boost_USE_STATIC_LIBS ON)
+find_package(Boost REQUIRED COMPONENTS system chrono date_time filesystem iostreams thread zlib)
+include_directories(${Boost_INCLUDE_DIRS})
```

Comment:
With the imported Boost:: targets, this shouldn't be needed anymore. Also, this could cause problems if e.g. LSL wants to import from lslboost but the include directories are polluted by `include_directories`.

Reply:
I think this is at the crux of one issue here. Without it I get
c:\users\david.medine\clean_lsl\labstreaminglayer\apps\labrecorder\recording.h(11): fatal error C1083: Cannot open in
clude file: 'boost/thread.hpp': No such file or directory [C:\Users\David.Medine\clean_lsl\labstreaminglayer\build\Apps
\LabRecorder\LabRecorder.vcxproj]